src/app/controller/services/jit/operations-event.service.ts

```diff
@@ -20,8 +20,16 @@ export class OperationsEventService {
     this.EVENT_DELIVERY_STATUS_URL = globals.config.uiSupportBackendURL + "/unofficial/timestamp-info";
   }
 
-  getTimestampInfoForTransportCall = (transportCallId: string): Observable<TimestampInfo[]> => this.httpClient.get<TimestampInfo[]>(this.EVENT_DELIVERY_STATUS_URL + "?transportCallID=" + transportCallId + "&limit=" + this.LIMIT)
-
+  getTimestampInfoForTransportCall = (transportCallId: string,portCallPart? : string): Observable<TimestampInfo[]> => { 
+    let httpParams = new HttpParams();
+            
+    if (portCallPart) {
+      httpParams = httpParams.set('portCallPart', portCallPart);
+    }
+    return this.httpClient.get<TimestampInfo[]>(this.EVENT_DELIVERY_STATUS_URL + "?transportCallID=" + transportCallId + "&limit=" + this.LIMIT, {
```

Comment:
`transportCallID` and `limit` should be set via the `HttpParams` now that we use that (rather than mixing between the two different styles)

Reply:
very well, will do 👍